.pre-commit-config.yaml

```diff
@@ -1,46 +1,58 @@
 repos:
-  - repo: local
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: check-symlinks
+      - id: destroyed-symlinks
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-toml
+      - id: check-ast
+      - id: check-added-large-files
+      - id: check-merge-conflict
+      - id: check-executables-have-shebangs
+      - id: check-shebang-scripts-are-executable
+      - id: detect-private-key
+      - id: debug-statements
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.12.0
     hooks:
       - id: isort
-        name: isort
-        entry: isort
-        files: "python/.*"
-        exclude: "python/build/.*"
-        language: python
-        types: [python]
-        additional_dependencies:
-          - isort==5.10.1
-
+        stages: [commit, push, manual]
+  - repo: https://github.com/pre-commit/mirrors-autopep8
+    rev: v2.0.2
```

Comment:
Can we keep v1.6.0 in this PR? We can upgrade in a latter PR when the CI servers also get upgraded

Reply:
Restored.